packages/react/src/ActionGroup/ActionGroup.test.tsx

```diff
@@ -0,0 +1,41 @@
+import { render } from '@testing-library/react'
+import { createRef } from 'react'
+import { ActionGroup } from './ActionGroup'
+import '@testing-library/jest-dom'
+
+describe('Action Group', () => {
+  it('renders', () => {
+    const { container } = render(<ActionGroup />)
+
+    const component = container.querySelector(':only-child')
+
+    expect(component).toBeInTheDocument()
+    expect(component).toBeVisible()
+  })
+
+  it('renders a design system BEM class name', () => {
+    const { container } = render(<ActionGroup />)
+
+    const component = container.querySelector(':only-child')
```

Comment:
If you add `role="group"` to the actiongroup component you can select by role in jest.